shared/src/background.js

```diff
@@ -154,6 +154,21 @@ browser.runtime.onMessage.addListener(async (data) => {
   }
 });
 
+browser.commands.onCommand.addListener(async (command) => {
+  // TODO: Remove this
+  console.log('==== File.method', command);
+
+  if (command === 'summarize-active-page') {
+    await browser.windows.create({
+      url: browser.runtime.getURL('src/summarize_result.html'),
+      focused: true,
+      width: 600,
+      height: 500,
+      type: 'popup',
+    });
```

Comment:
The alternative to opening a popup like this was to open a regular, full-width tab, make it a sidepanel extension, or an overlay in the existing page.

I didn't go with making it a regular full-width tab because it seemed too empty, but it didn't seem "bad", honestly.

I didn't go with the sidebar (seemed the most interesting, as a "sticky extension" version) because the API is fairly new and [Chromium/Firefox diverge on implementation a lot](https://github.com/w3c/webextensions/issues/387).

Finally, I didn't go with the overlay (what I initially thought of because it would look more integrated and with the context inline) because [it would require injecting content into the existing page](https://stackoverflow.com/a/69361046), thus invasive permissions, and more prone to breaking.